_sass/_layout.scss

```diff
@@ -641,3 +641,8 @@ img {
         float: right;
     }
 }
+
+/* Regionally restricted content */
+.geo-DE {
+    display: none;
```

Comment:
Edge case: We might need to show this for non-JS users with a `<noscript>`?

Reply:
@moollaza Oh, yes. That's a good point. I've updated the `default.html` template to pull in the geo-specific css file in a `<noscript>`.